pkgs/apps/xtb/default.nix

```diff
@@ -23,25 +35,38 @@ stdenv.mkDerivation rec {
     mkl
   ];
 
+  src = fetchFromGitHub  {
+    owner = "grimme-lab";
+    repo = pname;
+    rev = "v${version}";
+    sha256= "0fcf9f6y93aii907as25vmchfvdyzyrk0w7nqwyv1mjvab9a9acc";
+  };
+
   hardeningDisable = [
     "format"
   ];
 
-  postFixup = let
-    # programs that XTB might call.
-    binSearchPath = with lib; strings.makeSearchPath "bin" ([ ]
-      ++ lists.optional (turbomole != null) turbomole
-      ++ lists.optional (orca != null) orca
-      ++ lists.optional (turbomole != null) cefine
-    );
-  in  ''
+  postInstall = ''
+    mkdir -p $out/lib/pkgconfig
+    substitute ${xtbPkgConfig} $out/lib/pkgconfig/xtb.pc \
```

Comment:
Instead of creating a file in the nix-store first, I would write the file directly in the `postInstall` phase:
```
cat > $out/lib/pkgconfig/xtb.pc << EOF
...
EOF
```
Here you have shell substitution of `$out` automatically.